Cargo.toml

```diff
@@ -24,6 +24,9 @@ rust-version = "1.80"
 default = []
 experimental-ion-hash = ["digest", "experimental-reader-writer"]
 
+# Access location information of the input Ion from underlying buffer.
+experimental-location = []
```

Comment:
Do we want to release this as "experimental"? Can we call it `element-location` instead?

Reply:
Sure. I think it should be okay to move this out of experimental. I don't want to attach it with element though. What if we decide to keep it with `LazyValue` and add support for `Element` later. 
Maybe we can just say `source-location`?